[2025-03-07T11:10:14.956+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.create_postgres_table manual__2025-03-07T11:05:13.690220+00:00 [queued]>
[2025-03-07T11:10:14.969+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.create_postgres_table manual__2025-03-07T11:05:13.690220+00:00 [queued]>
[2025-03-07T11:10:14.969+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 6
[2025-03-07T11:10:14.986+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2025-03-07 11:05:13.690220+00:00
[2025-03-07T11:10:14.991+0000] {standard_task_runner.py:57} INFO - Started process 1624 to run task
[2025-03-07T11:10:14.994+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'create_postgres_table', 'manual__2025-03-07T11:05:13.690220+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator.py', '--cfg-path', '/tmp/tmpbewljwzw']
[2025-03-07T11:10:14.996+0000] {standard_task_runner.py:85} INFO - Job 52: Subtask create_postgres_table
[2025-03-07T11:10:15.036+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.create_postgres_table manual__2025-03-07T11:05:13.690220+00:00 [running]> on host 5588030d4cfa
[2025-03-07T11:10:15.092+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-07T11:05:13.690220+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-07T11:05:13.690220+00:00'
[2025-03-07T11:10:15.092+0000] {sql.py:274} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
[2025-03-07T11:10:15.099+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T11:10:15.106+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T11:10:15.108+0000] {sql.py:418} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        , parameters: None
[2025-03-07T11:10:15.114+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=create_postgres_table, execution_date=20250307T110513, start_date=20250307T111014, end_date=20250307T111015
[2025-03-07T11:10:15.166+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-07T11:10:15.187+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
